pairinteraction/unit_test/integration_test_referencedata.txt

```diff
@@ -1 +1 @@
 180 181 182 185 186 187 190 191 192 195 196 197 203 204 213 214 223 224 11 12 13 16 17 18 22 23 24 25 26 27 32 33 34 40 41 42 119 153 154 155 163 164 165 173 174 175 180 181 182 185 186 187 190 191 192 195 196 197 200 201 202 203 204 205 206 207 213 214 215 223 224 225 16 17 18 22 23 24 26 27 29 30 31 33 34 35 41 42 43 120 154 155 156 164 165 166 174 175 176 190 191 192 195 196 197 200 201 202 204 205 206 207 210 211 212 214 215 216 217 224 225 226 22 23 24 27 29 30 31 34 35 37 38 39 42 43 44 121 155 156 157 165 166 167 175 176 177 200 201 202 205 206 207 210 211 212 215 216 217 220 221 222 225 226 227 29 30 31 34 35 37 38 39 42 43 44 46 47 48 51 52 53 122 156 157 158 166 167 168 176 177 178 206 207 208 210 211 212 215 216 217 218 220 221 222 225 226 227 228 230 231 232 235 236 237 37 38 39 42 43 44 46 47 48 51 52 53 123 157 158 167 168 177 178 207 208 217 218 220 221 222 225 226 227 228 230 231 232 235 236 237 46 47 48 51 52 53 124 158 168 178 208 218 228 230 231 232 235 236 237 3 6 12 13 17 18 28 32 33 36 40 41 45 49 50 125 141 142 146 147 148 159 163 164 169 173 174 181 182 186 187 188 191 192 196 197 198 209 213 214 219 223 224 229 233 234 3 4 6 7 12 13 17 18 23 24 28 29 32 33 34 36 37 40 41 42 45 46 49 50 51 126 141 142 146 147 148 151 152 156 157 158 159 160 163 164 165 169 170 173 174 175 181 182 186 187 188 191 192 196 197 198 201 202 206 207 208 209 210 213 214 215 219 220 223 224 225 229 230 233 234 235 3 4 5 6 7 8 17 18 23 24 28 29 30 31 33 34 35 36 37 38 41 42 43 45 46 47 50 51 52 127 141 142 146 147 148 151 152 156 157 158 159 160 161 162 164 165 166 167 168 169 170 171 174 175 176 191 192 196 197 198 201 202 206 207 208 209 210 211 212 214 215 216 217 218 219 220 221 224 225 226 229 230 231 234 235 236 4 5 7 8 9 23 24 29 30 31 34 35 37 38 39 42 43 44 46 47 48 51 52 53 128 151 152 156 157 158 160 161 162 165 166 167 168 170 171 172 175 176 177 178 201 202 206 207 208 210 211 212 215 216 217 218 220 221 222 225 226 227 228 230 231 232 235 236 237 5 8 9 30 31 35 38 39 43 44 47 48 52 53 54 129 161 162 166 167 168 171 172 176 177 178 211 212 216 217 218 221 222 226 227 228 231 232 236 237 238 8 9 31 38 39 43 44 47 48 52 53 54 130 162 167 168 171 172 176 177 178 212 217 218 221 222 226 227 228 231 232 236 237 238 12 13 32 40 49 131 163 173 181 182 186 187 188 213 223 233 12 13 17 18 32 33 40 41 49 50 132 163 164 173 174 181 182 186 187 188 191 192 196 197 198 213 214 223 224 233 234 12 13 17 18 23 24 32 33 34 40 41 42 49 50 51 133 163 164 165 173 174 175 181 182 186 187 188 191 192 196 197 198 201 202 206 207 208 213 214 215 223 224 225 233 234 235 17 18 23 24 30 31 33 34 35 41 42 43 50 51 52 134 164 165 166 174 175 176 191 192 196 197 198 201 202 206 207 208 211 212 214 215 216 217 218 224 225 226 234 235 236 23 24 30 31 34 35 38 39 42 43 44 51 52 53 135 165 166 167 175 176 177 201 202 206 207 208 211 212 215 216 217 218 221 222 225 226 227 228 235 236 237 30 31 35 38 39 43 44 47 48 52 53 54 136 166 167 168 176 177 178 211 212 216 217 2
```

Comment:
Needs to be ignored for reformatting. Otherwise dumping new reference data will trigger reformatting.